Project.toml

```diff
@@ -33,7 +33,7 @@ LinearAlgebra = "37e2e46d-f89d-539d-b4ee-838fcccc9c8e"
 MLJModelInterface = "e80e1ace-859a-464e-9ed9-23947d8ae3ea"
 MLJScientificTypes = "2e2323e0-db8b-457b-ae0d-bdfb3bc63afd"
 MultivariateStats = "6f286f6a-111f-5878-ab1e-185364afe411"
-NearestNeighbors = "b8a86587-4115-5ab1-83bc-aa920d37bbce"
+#NearestNeighbors = "b8a86587-4115-5ab1-83bc-aa920d37bbce"
 StableRNGs = "860ef19b-820b-49d6-a774-d7a799459cd3"
```

Comment:
Not sure you can put comments in the toml file this way.

Are you getting a pkg conflict? These can be tricky to sort out and I'd be glad to take a look.

Reply:
Yeah, not sure why. At the moment the CI thing seems to be stuck, tomorrow if this fail I will just revert back to an old commit in that pile that I know works.